public/pages/AnomalyCharts/components/FeatureChart/FeatureChart.tsx

```diff
@@ -217,8 +238,8 @@ export const FeatureChart = (props: FeatureChartProps) => {
                   domainType={AnnotationDomainTypes.XDomain}
                   dataValues={getFeatureMissingDataAnnotations(
```

Comment:
nit: Confused what the featureMissingDataAnnotations are (could just need an explanation offline).

Reply:
It's annotations on the chart to show if a user has sparse / missing data on the feature chart, where feature results couldn't be generated due to no data (e.g., no data in 10-min period w/ an interval of 10 mins may lead to no features, therefore no anomaly results).